packages/contracts/contracts/core/ManagedService.sol

```diff
@@ -21,9 +21,15 @@ contract ManagedService is AccessControl, ReentrancyGuard {
      *
      * @param relayer  The address of the account that made the call.
      * @param to       The address of the remote contract.
+     * @param value    The value transferred from the caller to the destination address.
      * @param dataHash A keccak256 hash of the input data.
      */
-    event Called(address indexed relayer, address indexed to, bytes32 indexed dataHash);
+    event Called(
```

Comment:
I added an event argument the `msg.value`. Seems like a good idea to include this, although it's not necessary